tests/fixtures/custom/echo.yml

```diff
@@ -1,19 +1,26 @@
 before_download:
   - echo "This command is run before download."
-  - pwd
+  - |
+    echo "Using hook dir: ${CUSTOM_DIR}"
+    echo "Running hook in:" `pwd`
 download:
   - echo "This command is run for each package download."
   - |
+    echo "Using hook dir: ${CUSTOM_DIR}"
+    echo "Running hook in:" `pwd`
     echo "Downloading PKG: ${PKG}"
     if [ "${PKG}" != "" ]; then
         mkdir "${PKG}"
         touch "${PKG}/${PKG}.spec"
     fi
-  - pwd
 before_build:
   - echo "This command is run before build."
-  - pwd
+  - |
+    echo "Using hook dir: ${CUSTOM_DIR}"
+    echo "Running hook in:" `pwd`
```

Comment:
I would prefer

```
$(pwd)
```

than 

```
`pwd`
```
.

Because of https://github.com/koalaman/shellcheck/wiki/Sc2006

Could you change to it?
Other lines for \`pwd\` are also same.